.github/workflows/tests.yml

```diff
@@ -1,21 +1,30 @@
 name: Tests
 
-on: [push]
+on: push
 
 jobs:
   test:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-latest, ubuntu-latest, windows-latest]
+        os: [macos-latest, ubuntu-latest]
 
     steps:
       - uses: actions/checkout@v2
       - name: Set up Python 3
         uses: actions/setup-python@v1
         with:
           python-version: 3.8
-      - name: Install required dependencies
+      - name: Override database files (these need to be fixed since the test values depend on the version of the dataset used)
+        run: rm -rf megago/resources && mv functional_tests/resources/ megago/resources/
+      - name: Install required dependencies on ${{ matrix.os }}
+        if: matrix.os != 'windows-latest'
+        run: |
+          python3 -m pip install --upgrade pip
+          pip3 install -r requirements-dev.txt
+          pip3 install -U .
+      - name: Install required dependencies on ${{ matrix.os }}
+        if: matrix.os == 'windows-latest'
         run: |
           python -m pip install --upgrade pip
           pip install -r requirements-dev.txt
```

Comment:
Windows is not tested anymore, is it? Besides, this looks like a duplicate: step `Install required dependencies on ${{ matrix.os }}` exists twice.

Reply:
I still want to test Windows, I just need to split up the test workflows cause there are too many differences between running it on a *nix-based OS and Windows. I'll make sure to include that before I re-request a review from you :) Thanks for pointing this out. Both steps with the if are actually different (one uses `python3` and `pip3` commands, the other one `python` and `pip` as required by Windows)